can/bus.py

```diff
@@ -142,7 +236,7 @@ def flush_tx_buffer(self):
     def shutdown(self):
         """
         Called to carry out any interface specific cleanup required
-        in shutting down a bus.
+        in shutting down a bus. Calls :meth:`~can.BusABC.flush_tx_buffer`.
```

Comment:
I don't know the reason for this. Maybe remove it. Is there any interface using it?

Reply:
Well, `ixxat`, `kavaser`, `nican` and `vector` all implement `flush_tx_buffer()`. But none of the interfaces calls the method, at least none of the internal ones. I propose we remove the call to `flush_tx_buffer()` from `shutdown()` and remove the comment I added.

Reply:
See 6a4aa5b.